lib/mixpanel-node.js

```diff
@@ -148,16 +148,24 @@ var create_client = function(token, config) {
             }
             try {
                 body = JSON.parse(body);
-                if (body.status === 0) {
-                    error = new Error('Mixpanel REST API reported an error: ' + body.error);
-                    console.error(error);
-                    return callback(error);
-                }
             } catch (e) {
-                error = new Error('Mixpanel REST API returned invalid JSON response');
+                error = new Error('Mixpanel REST API returned an invalid JSON response');
+                console.error(error);
+                return callback(error);
+            }
+
+            if (!body || typeof body.status !== 'number') {
```

Comment:
do you know how much the status should be? instead of checking the type of and !== 0?